pkgs/development/python-modules/mcp/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  hatchling,
+  anyio,
+  httpx-sse,
+  httpx,
+  pydantic-settings,
+  pydantic,
+  sse-starlette,
+  starlette,
+  uvicorn,
+}:
+
+buildPythonPackage rec {
+  pname = "mcp";
+  version = "1.2.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-KwbH7OmNbqnmN5yqONdLQyOFwzj7Uwy4Lixw6nrdlPU=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    anyio
+    httpx-sse
+    httpx
+    pydantic-settings
+    pydantic
+    sse-starlette
+    starlette
+    uvicorn
+  ];
+
+  pythonImportsCheck = [ "mcp" ];
+
+  # no tests implemented
```

Comment:
What is this https://github.com/modelcontextprotocol/python-sdk/tree/main/tests then?

Reply:
Thanks for pointing this out ; i fixed this oversight